packages/app/src/components/App/index.js

```diff
@@ -66,6 +84,7 @@ RainbowFirebaseApp.defaultProps = {
     translations: {},
     children: null,
     reducers: {},
+    initialize: () => {},
```

Comment:
if there is a default value it means there a function that will execute by default, is that right?